server/repo/repo-core/src/main/java/org/eclipse/vorto/repository/service/IModelRepository.java

```diff
@@ -78,6 +88,17 @@
 	void checkin(String handleId, String author);
 	
 	/**
+	 * @pre All Models are validated for integrated checks. 
+	 * 
+	 * @post Models are stored into repository and sends notifications to watchers. 
+	 * 
+	 * Checks in multiple models into the repository.
+	 * @param uploadHandle - all model file upload with file handles.
+	 * @param author
+	 */
+	void checkinMultiple(ModelHandle[] modelHandles, String author);	
```

Comment:
Same as previous comment. Please use the existing checkin(string handle, string author) instead of introducing another interface method doing a similar thing.